packages/react-composites/src/composites/CallComposite/pages/TransferPage.tsx

```diff
@@ -0,0 +1,231 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import {
+  IPersonaStyleProps,
+  IPersonaStyles,
+  IStyle,
+  IStyleFunctionOrObject,
+  Spinner,
+  SpinnerSize,
+  Stack,
+  Text,
+  mergeStyles
+} from '@fluentui/react';
+import { ErrorBar, OnRenderAvatarCallback } from '@internal/react-components';
+import React, { useCallback, useLayoutEffect, useMemo, useRef, useState } from 'react';
+import { useLocale } from '../../localization';
+import { CallArrangement } from '../components/CallArrangement';
+import { usePropsFor } from '../hooks/usePropsFor';
+import { useSelector } from '../hooks/useSelector';
+import { getRemoteParticipants } from '../selectors/baseSelectors';
+import { toFlatCommunicationIdentifier } from '@internal/acs-ui-common';
+import { AvatarPersona, AvatarPersonaDataCallback } from '../../common/AvatarPersona';
+/* @conditional-compile-remove(call-transfer) */
+import { getTransferCall } from '../selectors/baseSelectors';
+import { reduceCallControlsForMobile } from '../utils';
+import { LobbyPageProps } from './LobbyPage';
+
+/* @conditional-compile-remove(call-transfer) */
+type TransferPageSubject = 'transferor' | 'transferTarget';
+
+/**
+ * @private
+ */
+export const TransferPage = (
+  props: LobbyPageProps & {
+    /** Render component function to replace the default Persona Icon representing the transferor or transfer target. */
+    onRenderAvatar?: OnRenderAvatarCallback;
+    /** Callback function that can be used to provide custom data to Persona Icon rendered */
+    onFetchAvatarPersonaData?: AvatarPersonaDataCallback;
+  }
+): JSX.Element => {
+  const errorBarProps = usePropsFor(ErrorBar);
+  const strings = useLocale().strings.call;
+  const remoteParticipants = useSelector(getRemoteParticipants);
+  /* @conditional-compile-remove(call-transfer) */
+  const transferCall = useSelector(getTransferCall);
+
+  // Reduce the controls shown when mobile view is enabled.
+  const callControlOptions = props.mobileView
+    ? reduceCallControlsForMobile(props.options?.callControls)
+    : props.options?.callControls;
+
+  /* @conditional-compile-remove(call-transfer) */
+  // page subject is which should be participant shown in the transfer page depending on the transfer call state
+  const pageSubject: TransferPageSubject = useMemo(() => {
```

Comment:
So at the very beginning, the page shows transferor then once we have the transferTarget, the page will show the transferTarget instead?

Reply:
Yes, that's right